scripts/bootstrap.sh.in

```diff
@@ -182,24 +254,48 @@ ${gpg_keys:+gpgkey=${gpg_keys%?}}
 EOF
 }
 
-install_salt_minion() {
-    local -a yum_opts=(
-        '--assumeyes'
-        '--disablerepo=*'
-        '--enablerepo=metalk8s-*'
-    )
-
-    "$YUM" install "${yum_opts[@]}" salt-minion
+configure_apt_local_repository() {
+    local -r repo_name=$1
+    local -r repo_path="$BASE_DIR/packages/debian/$repo_name"
+    echo "deb [trusted=yes] file://$repo_path bionic $repo_name" \
+        > /etc/apt/sources.list.d/"$repo_name".list
 }
 
-install_genisoimage() {
-    local -a yum_opts=(
-        '--assumeyes'
-        '--disablerepo=*'
-        '--enablerepo=metalk8s-*'
-    )
-
-    "$YUM" install "${yum_opts[@]}" genisoimage
+install_packages() {
+    local -ra packages=("$@")
+    local -a repo_list=(/etc/apt/sources.list)
+
+    case "$OS" in
+        centos)
+            local -a yum_opts=(
+                '--assumeyes'
+                '--disablerepo=*'
+                '--enablerepo=metalk8s-*'
+            )
+            "$YUM" install "${yum_opts[@]}" "${packages[@]}"
+            ;;
+        ubuntu)
+            mapfile -O 1 -t repo_list < <(find \
+                /etc/apt/sources.list.d/ \
+                ! -name 'metalk8s-*' -a -name '*.list')
+
+            for source_list in "${repo_list[@]}"; do
+                mv "$source_list"{,.disabled}
+            done
+            "$APT" update
+            DEBIAN_FRONTEND=noninteractive "$APT" -y install "${packages[@]}" \
+                || exit_code=$?
+
+            for source_list in "${repo_list[@]}"; do
+                mv "$source_list"{.disabled,}
+            done
+            "$APT" update
```

Comment:
You should not update after restoring the repositories as you don't know if system repositories are well configured.
If it's not the case, the bootstrap will fail here because `apt update` will return a non-zero exit code.
Plus, even if repositories are well configured, it'll take some time for something we don't care/need.